features/hugepages/examples/hugepages-machine-config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+    labels:
+        machineconfiguration.openshift.io/role: hugepages
+    name: hugepages
+spec:
+    kernelArguments:
```

Comment:
Just in case, I'd say the kernelArguments would be better if they are quoted (as in https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfiguration.md#kernelarguments )

Reply:
hm I think in general it does not really matter until you have some special characters under the value, but I do not have problems to add quotes under the PR.

Reply:
done